rpc/transaction_test.go

```diff
@@ -179,6 +179,8 @@ func TestTransactionReceipt(t *testing.T) {
 
 // TestGetTransactionStatus tests starknet_getTransactionStatus
 func TestGetTransactionStatus(t *testing.T) {
+	//TODO: implement a test case to 'failure_reason' before merge
```

Comment:
t.Skip("todo: implement test for rpcv08-rc0 when full nodes support this rpc version")